tests/next_tests/integration_tests/feature_tests/ffront_tests/test_execution.py

```diff
@@ -81,7 +81,7 @@ def testee(a: cases.IJKField) -> cases.IJKField:
 
 
 @pytest.mark.uses_unstructured_shift
-def test_unstructured_shift(unstructured_case):
+def test_unstructured_shift_indexed(unstructured_case):
```

Comment:
```suggestion
def test_unstructured_shift(unstructured_case):
```
Let's keep the simple name for this case.